### PR TITLE
Merged structure of M3 with Login code. Setup React front-end

### PR DESCRIPTION
**Pull request by rydwhelchel**
Merged login code with the aforementioned structure idea from group chat. Connected React to Flask.

### CODE REVIEW

File: resources/models.py [hunk 1]
```diff
@@ -18,16 +20,4 @@ def set_password(self, password):
     def check_password(self, password):
         return check_password_hash(self.password_hash, password)
 
-
```
Comment (MiaLinlinLu) [inline resources/models.py:21]: Not sure about it.
I used to use separate files, and there are circular import errors. Does this work for you

**Review by MiaLinlinLu — COMMENTED** [review]
not sure about moving out the models from the file. 
Does that work? if it works well, then it is good !

**Comment by MiaLinlinLu** [comment]
great

**Review by MiaLinlinLu — APPROVED** [review]
greaat

File: resources/models.py (hunk 1 above)
Comment (rydwhelchel) [reply resources/models.py:21]: Hey, yeah there are circular imports if you use 2 files. Luckily John helped by showing that if you use a third file, `app_setup.py`, you can avoid circular imports.

**Pull request merged**